services/AttendanceService.ts

```diff
@@ -38,7 +38,7 @@ export default class AttendanceService {
     return this.transactions.readWrite(async (txn) => {
       const event = await Repositories.event(txn).findByAttendanceCode(attendanceCode);
       if (!event) throw new NotFoundError('Oh no! That code didn\'t work.');
-      if (!event.isOngoing()) throw new UserError('You can only enter the attendance code during the event!');
+      if (!event.isAcceptingAttendances()) throw new UserError('This event is not currently accepting attendances!');
 
```

Comment:
Wasn't too sure how to phrase the error to explain that 'you can only check in to events at most 30 minutes before and at least 30 minutes after the event' without being too long, @sumeet-bansal do you think this is preferred to simply 'this event is not currently accepting attendances'?

Reply:
@shravanhariharan2 Would we be able to make this message a bit more specific, based on the current time compared to the event window? For example "This event hasn't started yet, please wait to check in." or "This event has ended and is no longer accepting attendances"? I'm definitely open to alternate wording as well.

Reply:
+1 on ^